MarvelApiClient/src/main/java/com/karumi/MarvelApiClient/MarvelError.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *   Copyright (C) 2015 Karumi.
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.karumi.marvelapiclient;
+
+import com.google.gson.annotations.SerializedName;
+
+class MarvelError {
+  @SerializedName("code") private String code;
+  @SerializedName("message") private String message;
+  @SerializedName("status") private String status = "";
+
+  public String getCode() {
+    return code;
+  }
+
+  public String getMessage() {
+    return message;
+  }
+
+  public String getStatus() {
```

Comment:
Default visibility in the class but not in the methods. Should we change this method visibility to ``default`` instead of ``public``?